python_tests/pytest_opinfos.py

```diff
@@ -726,20 +727,52 @@
 )
 shape_ops.append(broadcast_opinfo)
 
-broadcast_in_dim_opinfo = OpInfo(
+# NOTE: The constant version of broadcast_in_dim opinfo tests the "shape"
+# argument when a List of Constant Ints is used as an input.
+# The symbolic parameter list lists the argument as "Constant" because
+# otherwise an input is generated to attempt to supply the "shape" arg.
+broadcast_in_dim_constant_opinfo = OpInfo(
     lambda fd: fd.ops.broadcast_in_dim,
-    "broadcast_in_dim",
+    "broadcast_in_dim_constant",
     sample_input_generator=broadcast_in_dim_generator,
     error_input_generator=broadcast_in_dim_error_generator,
     reference=jax.lax.broadcast_in_dim,
     reference_type=ReferenceType.Jax,
     symbolic_parameter_list=(
         ArgumentType.Symbolic,
+        # This argument is purposely Constant even though the positional
+        # argument can also be symbolic.
         ArgumentType.Constant,
         ArgumentType.Constant,
     ),
 )
-shape_ops.append(broadcast_in_dim_opinfo)
+shape_ops.append(broadcast_in_dim_constant_opinfo)
+
+
+# NOTE: The symbolic version of broadcast_in_dim opinfo tests the "shape"
+# argument with a Vector generated from another operation like ops.shape.
```

Comment:
Just so I understand: it's still not possible to mix constants with symbolic scalars in the argument list right? For example I could not do
`broadcast_in_dim(tv1, [3, s0], [True, False])`?

Reply:
No, that is possible.  I just did not show it.